Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM sbtscala/scala-sbt:eclipse-temurin-jammy-19.0.1_10_1.8.2_3.2.2
+
+WORKDIR /d4-chat
+ADD . /d4-chat
+
+EXPOSE ${APP_PORT}
+
+CMD ["sbt", "run"]
```

Comment:
The final image weight will be huge - around 1GB.
We can use docker stages to reduce the final size in times - the final weight will be around 285Mb.

The Dockerfile might looks like this:
```
FROM sbtscala/scala-sbt:openjdk-17.0.2_1.8.0_3.2.1 as build
RUN mkdir -p /root/build/project
COPY ./ /root/build
WORKDIR /root/build
RUN sbt d4-chat/stage

FROM openjdk:11-jre-slim-bullseye
WORKDIR .
COPY --from=build /root/build/target/universal/stage/ /app/
CMD app/bin/main
EXPOSE ${APP_PORT}
```